src/FindStmtCost.h

```diff
@@ -0,0 +1,148 @@
+#ifndef HALIDE_FIND_STMT_COST_H
+#define HALIDE_FIND_STMT_COST_H
+
+#include "Error.h"
+#include "ExternFuncArgument.h"
+#include "Function.h"
+#include "IRVisitor.h"
+#include "Module.h"
+
+#include <unordered_map>
+
+namespace Halide {
+namespace Internal {
+
+constexpr int NORMAL_NODE_CC = 1;
```

Comment:
This is the style we use for preprocessor macros, and the scope is a bit broad. I suggest moving these inside StmtCost and changing them to NormalNodeDMC and similar, so that usage looks like StmtCost::NormalNodeDMC